lxd/operations.go

```diff
@@ -336,7 +336,7 @@ func operationCancel(ctx context.Context, s *state.State, projectName string, op
 	localOp, _ := operations.OperationGetInternal(op.ID)
 	if localOp != nil {
 		if localOp.Status() == api.Running {
-			_, err := localOp.Cancel()
+			err := localOp.Cancel()
```

Comment:
these seem to be only used for token cancellation, should we `localOp.Wait` here too?